locales/it.json

```diff
@@ -211,7 +211,15 @@
       "zero": "Nessun messaggio",
       "one": "Hai 1 messaggio",
       "other": "Hai {{count}} messaggi"
-    }
+    },
+    "tab": {
+      "all": "Tutti",
+      "deadlines":"Scadenze"
+    },
+    "headerTitle": "io.italia.it",
```

Comment:
@fpersico is this header specific for this screen (I doubt that) - this translation should not be under `messages`